CONTRIBUTING.md

```diff
@@ -1,29 +1,55 @@
-# Contributing to Grafana
+# Contribute to Grafana
 
 Thank you for your interest in contributing to Grafana! We welcome all people who want to contribute in a healthy and constructive manner within our community. To help us create a safe and positive community experience for all, we require all participants to adhere to the [Code of Conduct](CODE_OF_CONDUCT.md).
 
-This document is a guide to help you through the process of contributing to Grafana.
+This document is a guide to help you through the process of making technical contributions to Grafana. 
 
-## Become a contributor
+Whether you're a new contributer or a seasoned veteran we hope these resources help you connect with the community:
 
-You can contribute to Grafana in several ways. Here are some examples:
+Interact and be heard:
+- Forums: Do you have a problem, question, or curiosity? Visit our [forums](https://gra.fan/fromgithubtoforums) for a reservoir of knowledge- submit your own questions and answers!
+- Meetups: Craving in-person connections without the long journeys? [Join your local Grafana & Friends meetup group](https://gra.fan/githubtomeetup)! 
+- Community Slack: Eager for real-time connections with fellow users? Begin a conversation on [Slack](https://gra.fan/githubtoslack). 
+Learn:
+- YouTube: From getting started to exploring newer projects like Pyroscope and Beyla, the [Grafana YouTube channel](https://gra.fan/githubtoyoutube) has what you need to get started! 
+- Meetups: Join a [group near you](https://gra.fan/githubtomeetup) to learn from local experts and ask questions in real time. 
+Share your story:
+- Meetups and blogs: We’d love to feature your OSS Grafana Labs use case or story at an upcoming Grafana & Friends meetup or on the Grafana blog! Submit your idea [here](https://gra.fan/githubtocca) and we’ll connect with you on next steps if accepted. 
 
-- Contribute to the Grafana codebase.
-- Report and triage bugs.
-- Develop community plugins and dashboards.
-- Write technical documentation and blog posts, for users and contributors.
-- Organize meetups and user groups in your local area.
-- Help others by answering questions about Grafana.
+## Make technical contributions
+
+We welcome your technical contributions! Here are some examples:
+
+- Contribute to the Grafana codebase- check out these [help-wanted issues]((https://github.com/grafana/grafana/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22))
+- Develop community [plugins](https://grafana.com/developers/plugin-tools) 
+- Report [bugs](https://github.com/grafana/grafana/issues/new?template=0-bug-report.yaml)
+- [Triage issues](https://github.com/grafana/grafana/blob/4414b92e93440cc9ed0f281989ee71dc16216a15/contribute/triage-issues.md)
```

Comment:
not a problem, just fyi: there is an automation triaging issues already (in grafana/grafana) but there's no harm in linking to `triage-issues.md` as it lists a few other things that can be manually done to improve the triage process. So, all good 👍🏼